arviz/tests/test_plots.py

```diff
@@ -648,7 +648,7 @@ def test_plot_rank(models, kwargs):
         {"ref_val": None},
         {"ref_val": {"mu": [{"ref_val": 1}]}},
         {"bins": None, "kind": "hist"},
-        {"mu": {"ref_val": (-1, 1)}},
+        {"ref_val": {"mu": [{"ref_val": -1}, {"ref_val": 1}]}},

```

Comment:
I think this will fix the test (we'll see) but it does not plot 2 reference values, only the first. What was the goal of this test? It used to work because "mu" was a kwarg and kwargs were ignored for kde, they were only taken into account for hist.

Reply:
I decided to convert the test to testing multiple ref_vals with coordinates which was not tested before.